### PR TITLE
[blog] announce lts-22 and nightly ghc-9.8

### PR DESCRIPTION
**Pull request by juhp**

### CODE REVIEW

**Comment by juhp** [comment]
https://github.com/commercialhaskell/stackage-content/blob/ca38d51fb80cb9e125621b0a1f8d8e71857e69d8/posts/announce-lts-22-nightly-ghc-9.8.md

File: posts/announce-lts-22-nightly-ghc-9.8.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+title: LTS 22 release for ghc-9.6 and Nightly now on ghc-9.8
+description: Stackage LTS 22 (ghc-9.6.3) has been released and Stackage Nightly moves to ghc-9.8.1.
+timestamp: 2024-01-03T06:16:00Z
+---
+
+## Stackage LTS 22 has been released
+
+The Stackage team is happy to announce that [Stackage LTS version 22](https://www.stackage.org/lts-22.0) was released last month, based on GHC stable version 9.6.3.
+
+LTS 22 includes many [package changes](https://www.stackage.org/diff/lts-21.22/lts-21.0), and has over 3300 packages!
```
Comment (mpilgrem) [inline posts/announce-lts-22-nightly-ghc-9.8.md:12]: @juhp, this URL for the packages changes looks wrong to me.

Comment (alexeyzab) [reply posts/announce-lts-22-nightly-ghc-9.8.md:12]: Maybe it should be this one? https://www.stackage.org/diff/lts-22.0/lts-21.0

Comment (mpilgrem) [reply posts/announce-lts-22-nightly-ghc-9.8.md:12]: I think either that, or the one that compares the first of lts-22 with the last of lts-21 (lts-21.25) - the latter was the approach taken for the blog post announcing the lts-21 series.

Comment (juhp) [reply posts/announce-lts-22-nightly-ghc-9.8.md:12]: Thank you!  Great catch

**Comment by juhp** [comment]
Updated: [render](https://github.com/commercialhaskell/stackage-content/blob/3dda247c817b365dc7f2316c49cfc9dacaeea30c/posts/announce-lts-22-nightly-ghc-9.8.md)

**Comment by juhp** [comment]
Updated: [render](https://github.com/commercialhaskell/stackage-content/blob/53c6a51f1f262b18782bcbac00d0b269cf291e85/posts/announce-lts-22-nightly-ghc-9.8.md)

**Comment by juhp** [comment]
More tweaks: [render](https://github.com/commercialhaskell/stackage-content/blob/be832d66fb307c51401e99fc9e9eaf91ffc1b701/posts/announce-lts-22-nightly-ghc-9.8.md)

- add heading for HF server
- slack link
- nightly diff link
- mention build-constraints.yaml

**Comment by juhp** [comment]
[preview](https://github.com/commercialhaskell/stackage-content/blob/28227638ea6911556651a05a0241eea7811573f1/posts/announce-lts-22-nightly-ghc-9.8.md)

- link to build-constraints

**Review by mihaimaruseac — APPROVED** [review]
Should we use first names for everyone or full names? Right now, we're doing both.

**Comment by juhp** [comment]
> Should we use first names for everyone or full names? Right now, we're doing both.

Right - fair point, it was kinda intentional - I particularly wanted to call out Adam's work - but sure I can add full names :+1:

**Comment by juhp** [comment]
Updated

For preview: `Files changed` -> `...` -> `View`

**Pull request merged**